app/blueprints/assessments/templates/sub_criteria.html

```diff
@@ -45,6 +46,10 @@ <h2 class="govuk-error-summary__title">
     {{ flag_application_button(application_id) }}
 {% endif %}
 
+{% if migration_banner_enabled %}
```

Comment:
Instead of adding the migration banner in every HTML. Can we make this part of [banner_summary.html](https://github.com/communitiesuk/funding-service-design-assessment/blob/main/app/templates/macros/banner_summary.html) and just pass the `migration_banner_enabled` variable